shell/components/SortableTable/THead.vue

```diff
@@ -235,7 +235,11 @@ export default {
         :align="col.align || 'left'"
         :width="col.width"
         :class="{ sortable: col.sort, [col.breakpoint]: !!col.breakpoint}"
+        :tabindex="col.sort ? 0 : -1"
+        class="sortable-table-head-element"
         @click.prevent="changeSort($event, col)"
+        @keyup.enter="changeSort($event, col)"
+        @keyup.space="changeSort($event, col)"
```

Comment:
Table headers should also indicate sort order with `aria-sort`[^1], but it doesn't look like there's a readily available way access sort order from within the component..

[^1]: https://www.w3.org/WAI/ARIA/apg/practices/grid-and-table-properties/#gridAndTableProperties_sort

Reply:
Was able to create a function that give us the sort string according to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort#values